hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -135,7 +135,7 @@ def test_get_calibration_from(self):
 class Test_quantification:
 
     def setUp(self):
-        s = EDSTEMSpectrum(np.ones([2, 1024]))
+        s = EDSTEMSpectrum(np.ones([2, 2, 1024]))
         energy_axis = s.axes_manager.signal_axes[0]
```

Comment:
Not necessarily, it goes back to the two different definitions of dose again. Both are area independent values. However cross sections make use of a dose per unit area (hence the need for a pixel area). This essentially allows for standards samples to be take at a different magnification from the real samples.